near/omni-bridge/src/lib.rs

```diff
@@ -425,6 +401,57 @@ impl Contract {
             )
     }
 
+    fn init_transfer(
+        &mut self,
+        sender_id: AccountId,
+        amount: U128,
+        init_transfer_msg: InitTransferMsg,
+    ) -> U128 {
+        let token_id = env::predecessor_account_id();
```

Comment:
Let's pass the `token_id` as an argument

Reply:
Done 7aa8115d52d0c3a5f3c18c0954ddfeede6f6c83b